.circleci/config.yml

```diff
@@ -0,0 +1,127 @@
+version: 2
+
+defaults: &defaults
+  working_directory: ~/circleci-champaign-ak-processor
+  docker:
+    - image: soutech/ruby2.4.1_node_awscli:0.2
+
+jobs:
+  install_dependencies:
+    <<: *defaults
+    steps:
+      - checkout
+      - restore_cache:
+          name: Restoring bundler cache
+          key: gem-cache-{{ checksum "Gemfile.lock" }}
+      - run:
+          name: Install gems
+          command: bundle install --jobs 4 --deployment
+      - save_cache:
+          key: gem-cache-{{ checksum "Gemfile.lock" }}
+          paths:
+          - vendor/bundle
+      - persist_to_workspace:
+          root: .
+          paths:
+            - vendor/bundle
+            - .bundle
+
+  rspec:
+    docker:
+      - image: soutech/ruby2.4.1_node_awscli:0.2
+        environment:
+          PG_USERNAME: champaign
+          PG_PASSWORD: champaign
+          RAILS_ENV: test
+          AK_HOST: https://act.sumofus.org/rest/v1
+          AK_USERNAME: ak_username
+          AK_PASSWORD: ak_password
+          HPC_RULE_ID: 22
+          AK_SUBSCRIPTION_PAGE_EN: 'registration'
+          AK_SUBSCRIPTION_PAGE_DE: 'registration_german'
+          AK_SUBSCRIPTION_PAGE_FR: 'registration_french'
+          AWS_REGION: us-west-2
+          CONSTITUENCIES_TABLE: UkConstituencies
+      - image: circleci/postgres:9.4-alpine-ram
+        environment:
+          POSTGRES_USER: champaign
+          POSTGRES_DB: champaign_test
+      - image: redis
+    steps:
+      - checkout
+      - attach_workspace:
+          at: ~/circleci-champaign-ak-processor
+      - run:
+          name: Bundler
+          command: bundle --path vendor/bundle
```

Comment:
This should just find the dependencies from `vendor/bundle` - they were installed before and persisted to workspace